resources/views/components/news/carousel-item.blade.php

```diff
@@ -17,15 +17,15 @@ class="ease-out duration-[300ms] -z-1 h-full w-full object-cover opacity-50 grou
         class="py-6 px-8 absolute top-0 left-0 w-full h-full"
         style="background: linear-gradient(180deg, #00000066 20%, #ffffff01 100%)"
     >
-        <h4 class="shadowoutline lg:opacity-0 transition delay-100 duration-300 subpixel-antialiased {{ $index === 0 ? '!opacity-100' : '' }}">
+        <h4 class="text-shadow lg:opacity-0 transition delay-100 duration-300 subpixel-antialiased {{ $index === 0 ? '!opacity-100' : '' }}">
             <a href="{{ $news->Link }}">{{ $news->Title }}</a>
         </h4>
 
-        <p class="newstext shadowoutline lg:opacity-0 transition delay-300 duration-300 subpixel-antialiased {{ $index === 0 ? '!opacity-100' : '' }}">
+        <p class="text-neutral-100 text-shadow lg:opacity-0 transition delay-300 duration-300 subpixel-antialiased {{ $index === 0 ? '!opacity-100' : '' }}">
             {!! $news->Payload !!}
         </p>
 
-            <div class="hidden sm:flex flex-col newsauthor shadowoutline absolute bottom-2 right-2">
+            <div class="hidden sm:flex flex-col justify-end items-end text-shadow absolute bottom-2 right-2">
```

Comment:
It looks like my text shadows are missing. Do I need to do something more than `npm run build`?
![image](https://github.com/user-attachments/assets/84278356-8fc7-4fb5-97e6-0f4ebb1d6742)

Reply:
Good catch, fixed in latest.